packages/element-api/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:10.16
+
+WORKDIR /srv/api
+
+COPY --chown=node:node package.json package-lock.json ./
+
+RUN npm install --quiet
```

Comment:
Why quiet? Curious in case of npm install fail or outdated packages wouldn’t we want to see output?

Reply:
true true, that was in the example I followed but I agree with you